oak/server/rust/oak_runtime/src/node/grpc/server.rs

```diff
@@ -267,13 +265,12 @@ impl UnaryService<Vec<u8>> for GrpcRequestHandler {
             METRICS.grpc_requests_total.inc();
             let timer = METRICS.grpc_request_duration.start_timer();
 
-            // Decode a gRPC request.
-            let grpc_request =
-                encap_request(request.get_ref(), &handler.method_name).ok_or_else(|| {
-                    let warning = "Failed to parse an incoming Protobuf message";
-                    warn!("{}", warning);
-                    tonic::Status::new(tonic::Code::InvalidArgument, warning)
-                })?;
+            // Create a gRPC request.
+            let grpc_request = GrpcRequest {
+                method_name: handler.method_name.to_string(),
+                req_msg: request.into_inner(),
+                last: true,
```

Comment:
Perhaps add a TODO to support streaming? Is there already an issue for it?

Reply:
Added TODO
Created a tracking issue https://github.com/project-oak/oak/issues/953